KUBEFLOW-OUTREACH-COMMITTEE.md

```diff
@@ -0,0 +1,60 @@
+# Kubeflow Outreach Committee (KOC)
+
+The Kubeflow Outreach Committee (KOC) is a committee dedicated to fostering growth, engagement, and community outreach for the Kubeflow project. It will focus on activities that promote Kubeflow to new and existing users, contributors, and stakeholders while building an inclusive, vibrant, and diverse community within the broader AI/ML ecosystem.
+
+The governance of the Kubeflow Outreach Committee will evolve as the community grows, ensuring that outreach activities remain relevant and impactful.
+
+### Charter
+
+1. Promote the vision, values, and mission of the Kubeflow project.
+2. Lead efforts to grow and engage the AI/ML community, including organizing events, producing educational materials, and advocating for Kubeflow across various tech communities.
+3. Help grow the community by bringing new contributors and users to Kubeflow, particularly through accessible resources and community-building initiatives.
+4. Collaborate with other committees and working groups to align outreach efforts with the broader goals of the Kubeflow project.
+5. Maintain the Kubeflow Website, documentation, demonstration, and educational material for the Kubeflow community.
+
+### Committee Structure
```

Comment:
My suggestion is we have:
1. Chairs / Chief Marketing Officers  - appointed by KSC members
2. Leads - appointed by KOC members

WDYT @franciscojavierarceo @chasecadet @varodrig @thesuperzapper @juliusvonkohout @StefanoFioravanzo ?

Reply:
Agree

Reply:
I still strongly believe that appointment should be by majority vote of the KSC, not allowing a single KSC member to appoint at their discretion.

Reply:
Yeah I'm good with that. That was the intention, I'll say that explicitly:

Reply:
Chairs/leads sounds fine. I would avoid CMO though.

Reply:
>I would avoid CMO though.
+1

Reply:
I agree not to have a CMO because that's probably a strange title for an open source project (though I could be wrong here). 

I do think that social media and content are a part of marketing and so that should fall under the scope of outreach. I think the items you mention also fall under the scope of outreach, so I believe we're aligned there...which means "outreach" is a good naming convention, I think.